pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -1138,17 +1140,17 @@ func Test_Run_Positive_VolumeFSResizeControllerAttachEnabled(t *testing.T) {
 			// Start the reconciler to fill ASW.
 			stopChan, stoppedChan := make(chan struct{}), make(chan struct{})
 			go func() {
+				defer close(stoppedChan)
 				reconciler.Run(stopChan)
```

Comment:
this `defer` is interesting, in what case `reconciler.Run` would throw an error so that `close` is not called?

Reply:
I just think “defer” will avoid hanging here.

This defer is not related with the date race.

Reply:
I was just curious about the why, I believe that if there's an error in `reconciler.Run` the test would fail and cleanup at that point wouldn't affect the result of the test, I don't think that reverting this part is necessary, thanks for the explanation